src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/BreakpointComplications.java

```diff
@@ -377,8 +377,12 @@ public int hashCode() {
             result = 31 * result + dupSeqRepeatUnitRefSpan.hashCode();
             result = 31 * result + dupSeqRepeatNumOnRef;
             result = 31 * result + dupSeqRepeatNumOnCtg;
-            result = 31 * result + dupSeqStrandOnRef.hashCode();
-            result = 31 * result + dupSeqStrandOnCtg.hashCode();
+            for (final Strand strand : dupSeqStrandOnRef) {
+                result = 31 * result + Objects.hashCode( strand.ordinal() );
+            }
+            for (final Strand strand : dupSeqStrandOnCtg) {
+                result = 31 * result + Objects.hashCode( strand.ordinal() );
```

Comment:
Objects.hashCode(strand.ordinal()) is a very expensive way of calculating strand.ordinal().  Maybe just use strand.ordinal() here, and on lines 1081 and 1086 below.

Reply:
Thank you both! It's been done now.